src/main/java/com/home/MyWizardBot/controller/WebHookController.java

```diff
@@ -0,0 +1,24 @@
+package com.home.MyWizardBot.controller;
+
+
+import com.home.MyWizardBot.MyWizardTelegramBot;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.telegram.telegrambots.meta.api.methods.BotApiMethod;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+@RestController
+public class WebHookController {
+
+    private final MyWizardTelegramBot telegramBot;
+
+    public WebHookController(MyWizardTelegramBot telegramBot) {
```

Comment:
Bean injection through the constructor is correct. But since you are using `Lombok`, you can choose not to declare a constructor, but use annotation to generate it for required fields. This recommendation is optional, but it's up to you to use it or not.

Reply:
Removed the constructor, then used the annotation lombok @AllArgsConstructor